champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/axe/BullsCharge.java

```diff
@@ -61,8 +61,8 @@ public String[] getDescription(int level) {
         return new String[]{
                 "Right click with an Axe to activate",
                 "",
-                "Enter a rage, gaining <effect>Speed " + UtilFormat.getRomanNumeral(speedStrength + 1) + "</effect> for ",
-                "and giving <effect>Slowness " + UtilFormat.getRomanNumeral(slownessStrength + 1) + "</effect> to anything you hit for <stat>" + slowDuration + "</stat> seconds",
+                "Enter a rage, gaining <effect>Speed " + UtilFormat.getRomanNumeral(speedStrength) + "</effect> for <stat>" + speedDuration + "</stat>",
```

Comment:
should we include the time unit here? it will just say 'gaining Speed for 7 and giving...'

Reply:
yes, we should